### PR TITLE
moved expandable_fields to meta class

### LINKED ISSUE #6159: Move `expandable_fields` to meta class
We are using the old way, declaring `expandable_fields` at the class level instead of on the meta class.

Docs on https://github.com/rsinger86/drf-flex-fields

This file needs to be updated https://github.com/readthedocs/readthedocs.org/blob/master/readthedocs/api/v3/serializers.py

Ref https://github.com/readthedocs/readthedocs.org/pull/6158#discussion_r322472534

### PR DESCRIPTION
**Pull request by Iamshankhadeep**
fixes #6159

### CODE REVIEW

**Comment by Iamshankhadeep** [comment]
can anyone tell me why lint is failing?

File: readthedocs/api/v3/serializers.py [hunk 1]
```diff
@@ -215,14 +210,9 @@ class VersionSerializer(FlexFieldsModelSerializer):
...
 
-    expandable_fields = dict(
-        last_build=(
-            BuildSerializer,
-            dict(
-                source='last_build',
-            ),
-        ),
-    )
+    expandable_fields ={
```
Comment (saadmk11) [inline readthedocs/api/v3/serializers.py]:
```suggestion
    expandable_fields = {
```

File: readthedocs/api/v3/serializers.py [hunk 2]
```diff
@@ -448,17 +438,16 @@ class ProjectSerializer(FlexFieldsModelSerializer):
     created = serializers.DateTimeField(source='pub_date')
     modified = serializers.DateTimeField(source='modified_date')
 
-    expandable_fields = dict(
-        active_versions=(
-            VersionSerializer,
-            dict(
+    expandable_fields = {
+        'active_versions': (VersionSerializer,
```
Comment (saadmk11) [inline readthedocs/api/v3/serializers.py]:
```suggestion
        'active_versions': (
            VersionSerializer,
```

**Comment by Iamshankhadeep** [comment]
lint is failing because of the space and newline issues? @saadmk11

**Comment by saadmk11** [comment]
I think so. You can see the error log from here https://travis-ci.org/readthedocs/readthedocs.org/jobs/587389577

**Comment by Iamshankhadeep** [comment]
Sorry I do not know what is meta class can you provide me with some link that can help me to learn 'meta class'. Thanks.

**Comment by Iamshankhadeep** [comment]
@stsewd dont I have to rebase this pull request?

**Comment by stsewd** [comment]
@Iamshankhadeep no, it's fine.

**Comment by Iamshankhadeep** [comment]
okay @stsewd, Sir thank you a lot for helping me and @saadmk11 thanks for helping me understanding the lint error.

**Pull request merged**